sormas-base/setup/s2s-generate-cert.sh

```diff
@@ -53,73 +53,90 @@ else
   fi
 fi
 
-if [[ -z "${SORMAS_PROPERTIES}" ]]; then
-  DEFAULT_SORMAS_PROPERTIES_PATH="${ROOT_PREFIX}/opt/domains/sormas/sormas.properties"
-  if [[ -f "${DEFAULT_SORMAS_PROPERTIES_PATH}" ]]; then
-    SORMAS_PROPERTIES="${DEFAULT_SORMAS_PROPERTIES_PATH}"
+if [[ ! -d "${SORMAS_DOMAIN_DIR}" ]]; then
+  DEFAULT_SORMAS_DOMAIN_DIR="${ROOT_PREFIX}/opt/domains/sormas";
+
+  if [[ -d "${DEFAULT_SORMAS_DOMAIN_DIR}" ]]; then
+    SORMAS_DOMAIN_DIR="${DEFAULT_SORMAS_DOMAIN_DIR}";
   else
-    while [[ ! -f "${SORMAS_PROPERTIES}" ]]; do
-		  read -r -p "Please specify a valid sormas properties path: " SORMAS_PROPERTIES
+     while [[ ! -d "${SORMAS_DOMAIN_DIR}" ]]; do
+		  read -r -p "Please specify a valid sormas domain path: " SORMAS_DOMAIN_DIR
 	  done
-	  export SORMAS_PROPERTIES
   fi
+fi
+
+SORMAS_PROPERTIES="${SORMAS_DOMAIN_DIR}/sormas.properties"
+
+while [[ -z "${SORMAS_ORG_ID}" ]]; do
+  read -p "Please provide an ID for the organization: " SORMAS_ORG_ID
+done
+
+while [[ -z "${SORMAS_ORG_NAME}" ]]; do
+  read -p "Please provide the Name of your organization: " SORMAS_ORG_NAME
+done
+
+while [[ -z "${SORMAS_HOST_NAME}" ]]; do
+  read -p "Please provide the Hostname of the server: " SORMAS_HOST_NAME
+done
+
+read -p "Please provide the https port of the server (443): " SORMAS_HTTPS_PORT
```

Comment:
This should optionally be specified by environment variable. When it is 443, it should be ignored.

Reply:
By removing the import of ssl certificate there is no need to require a port.

The user can add it directly to the hostname if it's not 443.

I've added it because exporting the ssl certificate of host needed the hostname and the hostname with port too so I separated the host and port, but now I will remove the port as it is no more needed explicitly